examples/model_selection/plot_randomized_search.py

```diff
@@ -23,19 +23,20 @@
 import numpy as np
 
 from time import time
```

Comment:
The statement above that "
 The performance is slightly worse for the randomized search, though this
 is most likely a noise effect and would not carry over to a held-out test set." is not true in the PR's documentation build (https://24729-843222-gh.circle-artifacts.com/0/doc/auto_examples/model_selection/plot_randomized_search.html). But since we don't fix the `random_state` in the example, we should probably say something like "The performance may be"